patches/server/0005-Paper-config-files.patch

```diff
@@ -719,6 +719,7 @@ index 0000000000000000000000000000000000000000..52b02cb1f02d1c65b840f38cfc8baee5
 +        public boolean lagCompensateBlockBreaking = true;
 +        public boolean useDimensionTypeForCustomSpawners = false;
 +        public boolean strictAdvancementDimensionCheck = false;
++        public boolean predictShellOnRestart = true;
```

Comment:
I'm wondering whether it might be nice to have a system property for this instead of an option so that it can be set by hosts, though I doubt many of them use the restart command